javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,215 @@
+
+       /************************************************** 
+      *       1  :  Flight booking fullname function      *
+       ***************************************************/  
+
+           console.log('(1a) : Printing out full name by using returne \n\n ');
+
+     /*************************************************** 
+    * (1a) : Printing out full name by using return      *
+     ****************************************************/  
+
+
+
+
+var firstName='';
+let lastName='';
+function getFullName(firstName,lastName){
+  return firstName+' '+lastName;
+}
+let fullName=getFullName("Benjamin", "Hughes");
```

Comment:
const

Reply:
yea i will do const and i changed it here too